NETPROGRAMTest/BasicExerciseTest/Exercise05Test.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NETPROGRAM.BasicExercise;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+
+namespace NETPROGRAMTest.BasicExerciseTest
+{
+
+    [TestClass]
+    public class Exercise05Test
+    {
+
+        [TestMethod]
+        public void testCheckNumberRequest()
+        {
+            var test = new Exercise05();
+
+            Assert.AreEqual(test.checkNumberRequest(153), true);
+        }
+
+        [TestMethod]
+        public void testShowListNumber()
+        {
+            var test = new Exercise05();
+            var listTest = new List<int>() { 153 };
+
+            Assert.AreEqual(test.showListNumber(154).IndexOf(0), listTest.IndexOf(0));
```

Comment:
```suggestion
            Assert.AreEqual(test.showListNumber(154)[0], 153);
```

IndexOf là hàm đi tìm kiếm của 1 số, ở đây là tìm số 0, vì nó không có nên sẽ ra -1.

Reply:
done